package.json

```diff
@@ -14,36 +14,42 @@
     },
     "homepage": "https://github.com/osskit/dafka-consumer#readme",
     "scripts": {
-        "test:teardown": "docker-compose -f tests/kafka.yml down -v --remove-orphans && docker-compose -f tests/dafka.yml down -v --remove-orphans && docker-compose -f tests/target.yml down -v --remove-orphans",
-        "test:setup": "docker-compose -f tests/target.yml up -d && sleep 30 && docker-compose -f tests/kafka.yml up -d && sleep 30 && docker-compose -f tests/dafka.yml up --build -d && sleep 30",
-        "test:start": "jest --runInBand --forceExit --config tests/jest.config.local.js --testResultsProcessor jest-teamcity-reporter",
-        "test": "yarn && npm run test:teardown && npm run test:setup && npm run test:start && npm run test:teardown",
+        "pretest": "bazel build //src:image.tar && docker load -i bazel-bin/src/image.tar",
+        "test": "NODE_OPTIONS=--experimental-vm-modules jest --config tests/jest.config.ts",
         "format": "prettier --write '**/*.{ts,js,json,java}'",
         "logs": "docker-compose -f tests/dafka.yml logs"
     },
     "dependencies": {
+        "@osskit/wiremock-client": "^4.3.0",
         "@types/jest": "^24.0.12",
         "@types/lodash": "^4.14.149",
-        "@types/node": "^11.13.8",
+        "@types/node": "^18.16.3",
         "@types/node-fetch": "^2.5.2",
         "@types/uuid": "^3.4.4",
         "delay": "^4.2.0",
-        "jest": "^24.9.0",
-        "jest-cli": "^24.7.1",
-        "jest-teamcity-reporter": "^0.9.0",
+        "jest": "^29.3.1",
+        "jest-config": "^29.3.1",
         "kafka-node": "4.1.3",
         "lodash": "^4.17.15",
         "node-fetch": "^2.6.0",
         "simple-fake-server-server-client": "^2.0.1",
```

Comment:
remove it

Reply:
why? we need both I think

Reply:
we still use simple-fake-server-server-client after you added @osskit/wiremock-client?

Reply:
I didn't saw you talked about it, just removed it